### ISSUE TITLE
Utiliser l'ID dans l'URL d'un lieu

### ISSUE THREAD

**Issue by johanricher** [MTES-MCT/acceslibre#277]
## Description

Ce schéma, qui est utilisé pour "standardiser" les données [produites](https://acceslibre.beta.gouv.fr/a) et [publiées](https://www.data.gouv.fr/fr/datasets/accessibilite-des-etablissements-recevant-du-public-erp-pour-les-personnes-en-situation-de-handicap/) par le projet acceslibre, identifie les lieux à l'aide d'un champ `id`.

Exemple : le lieu Chez Valentin à Montauban
* sur le site son URL est `https://acceslibre.beta.gouv.fr/app/82-montauban/a/poissonnerie/erp/chez-valentin/`
* dans le jeu de données son id est `36f4f44f-acf3-4275-97b8-c33b49786f2f`

Comment faire la correspondance entre un lieu identifié par un id dans le jeu de données et un lieu sur le site ?

D'autres schémas, jeux de données, services web ont besoin d'identifier un lieu référencé par acceslibre mais le font plutôt avec l'URL du lieu sur le site, donc d'une manière non pérenne (car l'URL peut changer). Voir par exemple [le nouveau schéma des lieux de médiation numérique](
https://github.com/LaMednum/standard-mediation-num/blob/v0.1.0/schema.json#L373).

Le fait que le site expose les IDs comme des URI sur le web les rendra exploitables par des services tiers (Wikidata par exemple).

## Proposition

Utiliser l'ID acceslibre des lieux pour les identifier sur https://acceslibre.beta.gouv.fr/

Les URL seraient sous la forme `https://acceslibre.beta.gouv.fr/app/{id}/` ou `https://acceslibre.beta.gouv.fr/app/lieu/{id}/` au lieu de `https://acceslibre.beta.gouv.fr/app/{numero departement}-{commune}/a/{activite}/erp/{nom-du-lieu}/`

**Comment by ColinMaudry** [comment]
Effectivement, ce serait plus efficace. Si vous souhaitez continuer à exposer vos URL actuelles, par exemple parce qu'elles sont plus lisibles, vous pouvez configurer votre application pour que les URL avec `département/commune/activité/nom`redirigent vers l'URL `/id`.

**Comment by mlvernay** [comment]
Nous sommes OK sur le constat (du moins pour le dataset sans URL : `acceslibre.csv`).
Nous avons prévu de retravailler nos URLs d'établissement, on évoquera le pb de l'id. 
On pourrait également proposer une rechercher par `id` 

Merci pour la contribution.

**Comment by ColinMaudry** [comment]
> On pourrait également proposer une rechercher par id

Je ne pense pas que ça se substitue à une adresse contenant l'identifiant. Ce type d'URL porte la clé qui distingue l'objet de tous les autres, c'est assez puissant quand on agrège des données.

**Comment by mlvernay** [comment]
Il existe `https://acceslibre.beta.gouv.fr/uuid/<erp_uuid>/` qui redirige vers l'url de la page detail d'un ERP.
